app/routes/admin/groups/components/GroupMembersList.js

```diff
@@ -44,9 +44,12 @@ const GroupMembersList = ({
   filters,
 }: Props) => {
   const GroupMembersListColumns = (fullName, membership) => {
+    console.log(groupsById);
     const { user, abakusGroup } = membership;
     const performRemove = () =>
+      // eslint-disable-next-line no-restricted-globals
       confirm(
+        // eslint-disable-next-line react/prop-types
```

Comment:
What is this warning, and how do we fix it?

Reply:
This looks like a bug in eslint. They closed the [issue](https://github.com/yannickcr/eslint-plugin-react/issues/2330) but even though we have the newest version it still does not look like it has been fixed.

Reply:
Not an issue as we moved to ConfirmModalWithParent instead of confirm()